controller/controller.go

```diff
@@ -268,10 +270,16 @@ func (r *reconcileIngress) reconcileIngress(ctx context.Context, ing *networking
 
 func (r *reconcileIngress) cleanUp(ctx context.Context, ingName types.NamespacedName, ing *networkingv1.Ingress) (reconcile.Result, error) {
 	var result reconcile.Result
-	err := r.cleanupNetworkAPI(ctx, ingName)
-	if err != nil {
-		return result, err
+
+	if takeOverVIPName := ing.Annotations[config.TakeOverAnnotation]; takeOverVIPName != "" {
+		// We wont remove the VIP, cause we use take over
```

Comment:
This is where we could rollback the vip-request to its former pools as we've discussed previously, right?

Reply:
I figured out that the nginx-operator will replace this annotations in each reconcile loop, them, I prefer to do this manual guided by documentation.